src/main/solidity/com/ampnet/identityservice/contract/Issuer.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IIssuer.sol";
+
+contract Issuer is IIssuer {
```

Comment:
You don't need the contract implementation, an interface is enough to generate contract wrapper. I know that properties cannot be specified in the Solidity interfaces, but you can specify their getters. For example, putting `function owner() external view returns address` in the interface will expose the `address public owner` field. My suggestion the is to remove this file and use only `IIssuer` for contract wrapper.

Reply:
Check out my comment to similar remark above.